src/popup/searchModule.js

```diff
@@ -1,13 +1,20 @@
 import { elements } from './base';
-import { unicodeToString, providerLogos, addLoadMoreButton } from './helper';
+import {
+  unicodeToString, providerLogos, addLoadMoreButton,
+} from './helper';
 import { activatePopup } from './infoPopupModule';
 import { removeSpinner } from './spinner';
 // eslint-disable-next-line import/no-cycle
 import bookmarkImage from './bookmarkModule';
-import { showNotification, removeChildNodes, restoreInitialContent } from '../utils';
+import {
```

Comment:
Undo these changes. Your editor must be wrapping the lines on save.

Reply:
Okay